soc/arm/nuvoton_npcx/common/soc_pins.h

```diff
@@ -80,6 +93,23 @@ void npcx_pinctrl_mux_configure(const struct npcx_alt *alts_list,
  */
 void npcx_pinctrl_i2c_port_sel(int controller, int port);
 
+/**
+ * @brief Get the asserted status of PSL input pads
+ *
+ * @param i index of 'psl_in_pads' prop
+ * @return 1 is asserted, otherwise de-asserted.
+ */
+bool npcx_pinctrl_psl_input_asserted(int i);
+
+/**
+ * @brief Configure PSL input pads in psl_in_pads list
+ *
+ * Used to configure PSL input pads list from "psl_in_pads" property which
+ * detect the wake-up events and the related circuit will turn on core power
+ * supply (VCC1) from standby power state (ultra-low power mode) later.
+ */
+void npcx_pinctrl_psl_input_configure(void);
```

Comment:
Is the implementation for this in another PR?

Reply:
Yes. So far, Nuvoton is working on moving scfg sources from pinctrl driver to npcx soc folder. Please refer [PR32859](https://github.com/zephyrproject-rtos/zephyr/pull/32859). Once it was merged, the other PR for psl implementation will be submitted.

Reply:
I added the implementation for configuring PSL input via device-tree in the last CL of this PR. Once [PR32859](https://github.com/zephyrproject-rtos/zephyr/pull/32859) was merged, I will do rebasing.

Reply:
Rebasing is done.